controllers/automation/policyautomation_controller.go

```diff
@@ -175,20 +175,26 @@ func (r *PolicyAutomationReconciler) Reconcile(
 			)
 
 			return reconcile.Result{RequeueAfter: requeueAfter}, nil
-		} else if policyAutomation.Spec.Mode == "once" {
-			log := log.WithValues("mode", "once")
+		} else if policyAutomation.Spec.Mode == policyv1beta1.Once {
+			log := log.WithValues("mode", string(policyv1beta1.Once))
 			targetList := common.FindNonCompliantClustersForPolicy(policy)
 			if len(targetList) > 0 {
 				log.Info("Creating an Ansible job", "targetList", targetList)
 
-				err = common.CreateAnsibleJob(policyAutomation, r.DynamicClient, "once", targetList)
+				err = common.CreateAnsibleJob(
```

Comment:
It'd be better if `CreateAnsibleJob` accepted a `PolicyAutomationMode` type instead so you don't have to convert to a string.

Reply:
I thought about this, it seems we still have `manual` and `scan` not in PolicyAutomationMode. Do you prefer we also add them in PolicyAutomationMode or keep `CreateAnsibleJob` mode parameter as a string?

Reply:
Ah, I see your point. Let's leave it as is.